### ISSUE TITLE
Move t=0 output (for INSTANT output) to its own file

### ISSUE THREAD

**Issue by bartgol** [E3SM-Project/scream#2974]
Talking offline with @rljacob , I found out that the way we save INSTANT output is not the preferred way. We currently save t=0 in the first set of files.

Solution: if the output file has storage_type=NumSnaps, save t=0 in a file with capacity 1snap, and close it. If the storage type is "Monthly" or "Yearly", I would still keep t=0 in the same file as other snaps.

**Comment by whannah1** [comment]
likely related to #2968

**Comment by rljacob** [comment]
I think t=0 is not included in an average (in EAM).  So if its output at all, its instant and in its own file.

If you want an average every 25 steps, the first one should be t=1 through t=25.

**Comment by bartgol** [comment]
> I think t=0 is not included in an average (in EAM). So if its output at all, its instant and in its own file.
> 
> If you want an average every 25 steps, the first one should be t=1 through t=25.

Yes, that's already the case in EAMxx. We only process t=0 if the avg type is INSTANT.
